options/options.go

```diff
@@ -135,6 +138,7 @@ func NewTerragruntOptions(terragruntConfigPath string) (*TerragruntOptions, erro
 		Sleep:                  DEFAULT_SLEEP,
 		RetryableErrors:        util.CloneStringList(RETRYABLE_ERRORS),
 		ExcludeDirs:            []string{},
+		IncludeDirs:            []string{},
```

Comment:
Please update the `Clone` method further down in this file too

Reply:
Done